utils/generator/README.md

```diff
@@ -0,0 +1,2245 @@
+# Proxy Library RPC Generator
```

Comment:
Can we make the readme file more concise similar to how the iOS one?
https://github.com/vladmu/sdl_ios/blob/SDL-0234-Proxy-Library-RPC-Generation/generator/README.md

It is currently very lengthy and it will be hard to keep it updated in the future whenever we make an update to the generator.

Reply:
@bilal-alsharifi it is lengthy due to numerous full examples with big copyright comments included in the document, will the removing of those examples solve the problem?

Reply:
If we remove all full examples as @vladmu suggested, README size will decrease 3 times, from 2450 lines to only 850 lines which is the same as we have in sdl_ios. @bilal-alsharifi please confirm if this looks good for you.

Reply:
I think samples are important to explain the rules. iOS also has sample codes. We can use fewer samples, pick classes with fewer properties as samples, and remove the license header form the samples in the readme to make the readme file shorter and easier to maintain. Please use the iOS one as a reference:
https://github.com/vladmu/sdl_ios/tree/SDL-0234-Proxy-Library-RPC-Generation/generator

Reply:
@bilal-alsharifi that iOS document was based on this Java document, and it just does not include the full examples, almost all other sections and their order are similar. So it is hard to understand what you want to change here. I've proposed above to remove FULL examples from the document, all SHORT samples stay in place. Please review simplified readme @ksologubov provided [here (7da2be7)](https://github.com/smartdevicelink/sdl_java_suite/blob/7da2be78ce3d0b0e6e2381e33d972cabcd9377ec/utils/generator/README.md)  and if that doesn't fit your expectation, please provide us more details about what you would like to change exactly. Thank you!

Reply:
Thanks, @vladmu for the details. I looked at the android simplified readme. I don't see any generated java sample code at all in it. I see snippets from the templates but not java code. However, in iOS, there are some full but concise samples like [this](https://github.com/vladmu/sdl_ios/tree/SDL-0234-Proxy-Library-RPC-Generation/generator#request-functions-sdlrpcrequest) for example. We just would like the readme to be easy to read but not extremely long.
As I suggested earlier, using fewer samples, picking classes with fewer properties as samples, and removing the license header from the samples may reduce the size of the file but still keep it self contained.

Reply:
Like for example, before today's updates, the readme had [three](https://github.com/smartdevicelink/sdl_java_suite/tree/744d2bfd76beb8f9adcc3158dca176ef05da270a/utils/generator#below-is-the-full-example-of-the-struct-class-with-simple-and-enum-parameters-inside) lengthy full sample for the `<struct>` section. After the update, they are all removed. I think having one concise sample that covers the basic cases will be a good balance between the two cases.

Reply:
Ok, got it, thank you, will update with basic samples

Reply:
@bilal-alsharifi Hi Bilal, I updated README with samples, could you please check

Reply:
Thank you. I will review that.